sdk/python/tests/integration/feature_repos/universal/online_store/remote.py

```diff
@@ -0,0 +1,89 @@
+
+from multiprocessing import Process
+from typing import Dict
+
+from feast import FeatureStore
+from feast.wait import wait_retry_backoff
+from tests.integration.feature_repos.integration_test_repo_config import IntegrationTestRepoConfig
+from tests.utils.http_server import check_port_open, free_port
+from tests.integration.feature_repos.universal.online_store_creator import (
+    OnlineStoreCreator,
+)
+
+class RemoteOnlineStoreCreator(OnlineStoreCreator):
+    _FEAST_SERVER_TIMEOUT_SEC:int = 30
+    def __init__(self, project_name: str, **kwargs):
+        super().__init__(project_name)
+        
+        for key, value in kwargs.items():
```

Comment:
Not a deal-breaker, but we should probably change `OnlineStoreCreator` similar to `DataSourceCreator`, meaning add a `setup` method that's called with a registry config.